.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,53 @@
+name: End to End integration tests
+
+on:
+  - pull_request
+  - push
+  - workflow_dispatch
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout rucio containers repository
+        uses: actions/checkout@v2
+        with:
+          repository: rucio/containers
+      - uses: actions/checkout@v2
+        name: Checkout rucio source
+        with:
+          path: dev/rucio
+      - name: Prepare Images
+        shell: bash
+        run: |
+          sed -i "s/RUN git clone .*/COPY .\/rucio \/tmp\/rucio/" $GITHUB_WORKSPACE/dev/Dockerfile_py3
+      - name: Build Images
```

Comment:
Moving the build and push (of the container images) to the `imagecache` workflow and instead just pulling and re-building here would resolve the issue with the PR not being able to run this testsuite.
Since the run is currently failing here, I am assuming that it will fail with any new PR, which makes this testsuite less useful. Without pushing the image to the GitHub image repository in this workflow it should work fine like the existing test workflows.